addon/components/tooltip-on-parent.js

```diff
@@ -11,16 +11,22 @@ export default Ember.Component.extend({
     return Ember.String.htmlSafe('display:none;');
   }),
 
-  registerOnParent: on('didInsertElement', function() {
-    const parentView = this.get('parentView');
+  init(...args) {
+    this._super(args);
 
-    if (parentView.renderTooltip) {
-      parentView.renderTooltip(this);
-    } else {
-      warn('No renderTooltip method found on the parent view of the {{tooltip-on-parent}} component');
-    }
+    run.schedule('afterRender', () => {
+      const parentView = this.get('parentView');
+      const componentNotInDOM = get(parentView, 'tagName') ? false : true;
+      if(componentNotInDOM) {
```

Comment:
Yeah it was just to be more descriptive about why we care about `tagName` but I'm happy to do it your way. I'll change that now and fix unit testing error.

BTW, sorry for the number of typos and UT failures, I'm in running fast mode and most my edits came from within Github :p